### PR TITLE
update window.web3 references

### PR DESCRIPTION
**Pull request by andrewdamelio**
fixes
- https://github.com/AugurProject/augur/issues/7673#issuecomment-627791102
- https://github.com/AugurProject/augur/issues/7673#issuecomment-627791401

### CODE REVIEW

**Review by bthaile — APPROVED** [review]
lgtm

File: packages/augur-ui/src/modules/auth/actions/login-with-injected-web3.ts [hunk 1]
```diff
@@ -101,7 +102,7 @@ const login = (account: string) => (
 ) => {
   const useGSN = getState().env['gsn']?.enabled;
   const provider = getWeb3Provider(windowRef);
-  const networkId = windowRef.web3.currentProvider.networkVersion;
+  const networkId = windowRef.web3?.currentProvider?.networkVersion || getState().env['networkId'];
```
Comment (MicahZoltu) [inline packages/augur-ui/src/modules/auth/actions/login-with-injected-web3.ts:105]: I recommend getting the network ID from `eth_chainId` (https://eips.ethereum.org/EIPS/eip-695).  This is the standard (for almost a year) way of getting the chain ID from the Ethereum provider.

File: packages/augur-ui/src/modules/auth/actions/login-with-injected-web3.ts [hunk 2]
```diff
@@ -36,7 +37,7 @@ export const loginWithInjectedWeb3 = () => async (
 
     const web3 = windowRef.web3;
 
-    if (web3.currentProvider?.publicConfigStore?.on) {
+    if (web3 && web3.currentProvider?.publicConfigStore?.on) {
```
Comment (MicahZoltu) [inline packages/augur-ui/src/modules/auth/actions/login-with-injected-web3.ts:40]: You can do null chaining here instead:
```ts
web3?.currentProvider?.publicConfigStore?.on
```

File: packages/augur-ui/src/modules/auth/actions/login-with-injected-web3.ts [hunk 3]
```diff
@@ -11,6 +11,7 @@ import {
   SIGNIN_LOADING_TEXT,
   SIGNIN_SIGN_WALLET,
   MODAL_LOADING,
+  NETWORK_IDS,
```
Comment (MicahZoltu) [inline packages/augur-ui/src/modules/auth/actions/login-with-injected-web3.ts:14]: This import doesn't appear to be used?

**Pull request merged**